src/main/java/org/jabref/gui/preferences/linkedfiles/LinkedFilesTabViewModel.java

```diff
@@ -54,16 +54,9 @@ public LinkedFilesTabViewModel(DialogService dialogService, PreferencesService p
         mainFileDirValidator = new FunctionBasedValidator<>(
                 mainFileDirectoryProperty,
                 mainDirectoryPath -> {
-                    ValidationMessage error = ValidationMessage.error(String.format(
-                            "%s%n%s%n%n%s%n%n%s > %s > %s",
-                            Localization.lang("Main directory not found"),
-                            mainDirectoryPath,
-                            Localization.lang("Please select a valid main directory under"),
-                            Localization.lang("Linked files"),
-                            Localization.lang("File directory"),
-                            Localization.lang("Main file directory")
-                    ));
-
```

Comment:
Why the hell did I do this...

Reply:
Re-use of existing translations I guess.

Good idea though. I try to re-use translations by grouping similar translations next to each other. Then, at the review on GitHub (by @calixtus e.g.), it can come up. Other than that, I hope that Crowdin also uses sub strings for its "brain" suggestions.

Reply:
Idea was to have just one common warning message and the already existing translations of the titles of the tabs and sections as breadcrumbs to find the preference setting.

Reply:
Otherwise you will have to create new translations for every validation message.

Reply:
> Idea was to have just one common warning message

I think, the old message was misleading, because one cannot touch the library preferences.

> and the already existing translations of the titles of the tabs and sections as breadcrumbs to find the preference setting.

There (IMHO) already was another message with "Linked files". I adapted that to be consistent.

Finally, I think, the preference is easy to find (we also have the search functionality). Thus, I like the short pointer to the preference.